oe_theme.theme

```diff
@@ -1652,3 +1490,28 @@ function oe_theme_theme_suggestions_block_alter(array &$suggestions, array $vari
   $suggestions[] = 'block__region_' . $block->getRegion();
   $suggestions[] = 'block__region_' . $block->getRegion() . '__' . $variables['elements']['#id'];
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function oe_theme_theme_suggestions_oe_corporate_blocks_ec_footer_alter(array &$suggestions, array $variables) {
+  $branding = theme_get_setting('branding') ?? 'core';
+  $suggestions[] = 'oe_corporate_blocks_ec_footer__' . $branding;
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function oe_theme_theme_suggestions_oe_corporate_blocks_eu_footer_alter(array &$suggestions, array $variables) {
+  $branding = theme_get_setting('branding') ?? 'core';
+  $suggestions[] = 'oe_corporate_blocks_eu_footer__' . $branding;
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_theme_preprocess_oe_corporate_blocks_eu_footer(array &$variables) {
```

Comment:
:copyright:  :spaghetti: ... also test didn't catch this, so I guess there should also be something to fix there.